From 966ef7e0b035625a5ca07754b60670e62f263dd8 Mon Sep 17 00:00:00 2001
From: Rakesh Babu <rsaladi2@marvell.com>
Date: Tue, 3 Aug 2021 10:55:16 +0530
Subject: [PATCH 1894/1921] octeontx2-vf: Export symbols needed by VF driver.

Exporting the required APIs in otx2_flows.c for VF driver to build
successfully as module. Without these changes there will be compilation
errors when trying to build VF driver as a module.

Change-Id: Ibcf479bf726f40fc1dd5e106e5a23a80b7a670d0
Signed-off-by: Rakesh Babu <rsaladi2@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/57974
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_flows.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_flows.c b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_flows.c
index 5ed3b2540982..f1cfc0edc4fc 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_flows.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_flows.c
@@ -261,6 +261,7 @@ int otx2vf_mcam_flow_init(struct otx2_nic *pfvf)
 
 	return 0;
 }
+EXPORT_SYMBOL(otx2vf_mcam_flow_init);
 
 int otx2_mcam_flow_init(struct otx2_nic *pf)
 {
@@ -309,6 +310,7 @@ void otx2_mcam_flow_del(struct otx2_nic *pf)
 {
 	otx2_destroy_mcam_flows(pf);
 }
+EXPORT_SYMBOL(otx2_mcam_flow_del);
 
 /*  On success adds mcam entry
  *  On failure enable promisous mode
-- 
2.31.1

